src/views/dataset/MultiSourceConfiguration.vue

```diff
@@ -250,17 +308,19 @@ export default class NewDataset extends Vue {
 
     this.maxFramesPerItem = 1;
     this.areStridesSetFromFile = false;
-    tiles.forEach((tile: any) => {
+    tiles.forEach(tile => {
       const frames: number = tile.frames?.length || 1;
       this.maxFramesPerItem = Math.max(this.maxFramesPerItem, frames);
       if (tile.IndexStride) {
-        this.assignmentNames.forEach(dimension => {
-          const stride = tile.IndexStride[`Index${dimension}`];
-          if (stride && stride > 0) {
-            this.strides[dimension] = stride;
-            this.areStridesSetFromFile = true;
+        Object.keys(this.dimensionNames).forEach(
+          (dimension: string | number) => {
```

Comment:
dimensionNames = { XY: "Positions", Z: "Z", T: "Time", C: "Channels" };

Object.keys(dimensionNames) returns an Array of strings, so I guess _dimension_ type should only be "string".

Reply:
Already fixed in another branch